android/quest/src/main/assets/configs/app/registers/practitioner_register_config.json

```
@@ -117,7 +117,8 @@
                   }
                 }
 
-              ]
+              ],
+
```

please revert, we don't use terminal commas in other places. can you else delete the empty line on 119, let's cleanup to remove any existing extra line breaks, thank you
```suggestion
              ]
```